devito/types/sparse.py

```diff
@@ -140,6 +225,99 @@ def _dist_datamap(self):
         """
         return self.grid.distributor.glb_to_rank(self._support) or {}
 
+    @cached_property
+    def dist_origin(self):
```

Comment:
group with public attributes

Reply:
All is ordered

_(cls
property
cached_property

Reply:
this one appearts to be in the middle between two private properties, `_dist_datamap` and `_pos_symbols`